### PR TITLE
Splitting GML tests into boost and qt part

### PR DESCRIPTION
**Pull request by rinkk**
this should solve #1023

### CODE REVIEW

File: Tests/CMakeLists.txt [hunk 1]
```diff
@@ -15,12 +16,8 @@ APPEND_SOURCE_FILES(TEST_SOURCES MeshLib)
 APPEND_SOURCE_FILES(TEST_SOURCES MeshGeoToolsLib)
 APPEND_SOURCE_FILES(TEST_SOURCES NumLib)
 
-set(TEST_SOURCES ${TEST_SOURCES}
-	FileIO/TestGLIReader.cpp
-	FileIO/TestCsvReader.cpp
-)
 if(QT4_FOUND)
-	set(TEST_SOURCES ${TEST_SOURCES} FileIO/TestXmlGmlReader.cpp)
+	set(TEST_SOURCES ${TEST_SOURCES} FileIO_Qt/TestQtGmlInterface.cpp)
```
Comment (bilke) [inline Tests/CMakeLists.txt]: Just use `APPEND_SOURCE_FILES` here as well (for consistency and the case there will be more test files in the future).

**Comment by TomFischer** [comment]
Is it possible to use exact the same 'FileIOGml' class in both tests? At least the surface test seems to be different.

**Comment by rinkk** [comment]
@TomFischer : No idea, Lars also didn't know a way to do this...

**Comment by bilke** [comment]
Jenkins test this please!

**Comment by TomFischer** [comment]
The classes `FileIOBoostGml` and `FileIOQtGml` are (almost?) the same. I would extract the class and reuse it in both tests. This would save ~200 lines.

**Comment by rinkk** [comment]
moved identical classes to header

**Comment by TomFischer** [comment]
:fast_forward:

**Pull request merged**